app/controllers/v1/audits_controller.rb

```diff
@@ -17,16 +17,8 @@ def show
 
     def create
       collection_policy.new(current_user).authorize! :new?
+      RepositoryAuditJob.perform_later(user: current_user)
```

Comment:
OK. Last, chance, ActiveJob...

If we're trying to preserve the ability to start an audit through the API, and the RESTful nature, we need the job to create the audit, enqueue the package jobs, and return the audit instance, so we can redirect to it with a 201.

If we can't get it to cooperate by switching this to a `perform_now` and giving us a real return value, it's back to a standalone class.